### PR TITLE
targets: zephyr: Follow Zephyr 1.7-pre on console refactors.

### PR DESCRIPTION
**Pull request by pfalcon**
JerryScript-DCO-1.0-Signed-off-by: Paul Sokolovsky paul.sokolovsky@linaro.org

### CODE REVIEW

**Comment by pfalcon** [comment]
@poussa , @sergioamr, @grgustaf : FYI

File: targets/zephyr/src/getline-zephyr.c [hunk 1]
```diff
@@ -15,19 +15,20 @@
 
 #include <zephyr.h>
 #include <uart.h>
+#include <drivers/console/console.h>
 #include <drivers/console/uart_console.h>
```
Comment (zherczeg) [inline targets/zephyr/src/getline-zephyr.c]: Do we need this uart_console.h include after the change?

Comment (pfalcon) [inline targets/zephyr/src/getline-zephyr.c]: Yes. Rest assured that I checked that ;-).

**Comment by poussa** [comment]
LGTM

**Pull request merged**